perf/benchmark/runner/runner.py

```diff
@@ -365,19 +421,26 @@ def run(args):
             ingress=args.ingress,
             mode=args.mode,
             mesh=args.mesh,
-            telemetry_mode=args.telemetry_mode,
-            cacert=args.cacert)
+            telemetry_mode=args.telemetry_mode)
 
     if fortio.duration <= min_duration:
         print("Duration must be greater than {min_duration}".format(
             min_duration=min_duration))
         exit(1)
 
-    for conn in fortio.conn:
-        for qps in fortio.qps:
-            fortio.run(conn=conn, qps=qps,
-                       duration=fortio.duration, size=fortio.size)
+    # Create a portmapping so we can access nighthawk_service.
+    popen_cmd = "kubectl -n \"{ns}\" port-forward svc/fortioclient {port}:9999".format(
```

Comment:
Aah: by portmap you mean port forward .  I was thinking nfs like portmapper.

Reply:
correct, this was bad naming, corrected that.